.ci/cleanup-packages.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace PhalconCi;
+
+class PackageCloud
+{
+    protected const API_ENDPOINT = "packagecloud.io";
+
+    private $apiKey;
+    private $user;
+    private $repo;
+
+    public function __construct($apiKey, $user, $repo)
+    {
+        $this->apiKey = $apiKey;
+        $this->user = $user;
+        $this->repo = $repo;
+    }
+
+    protected function buildRequestUrl($url)
+    {
+        $baseUrl = "https://" . $this->apiKey . ":@"  . self::API_ENDPOINT;
+        return $baseUrl . $url;
+    }
+
+    protected function getCurl()
+    {
+        $ch = curl_init();
+        curl_setopt($ch, CURLOPT_TIMEOUT, 30);
+        curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
+
+        return $ch;
+    }
+
+    public function query($url)
+    {
+        echo "Opening: " . $url . PHP_EOL;
+
+        $ch = $this->getCurl();
+
+        curl_setopt($ch, CURLOPT_URL, $this->buildRequestUrl($url));
+
+        if (false === ($retval = curl_exec($ch))) {
+            print_r(curl_error($ch));
+        } else {
```

Comment:
Can you close `$ch` somewhere here with `curl_close`

Reply:
Good catch. Added it.